ci/src/main/java/com/group21/ci/ContinuousIntegrationServer.java

```diff
@@ -25,6 +25,10 @@
  */
 public class ContinuousIntegrationServer extends AbstractHandler
 {
+    /**
+     * Constructs a ContinuousIntegrationServer object.
+     */
+    public ContinuousIntegrationServer(){};
     /**
      * Handler to handle incoming HTTP requests. The switch statement handles POST requests
      * from the GitHub push webhook and GET requests to display build history.
```

Comment:
Line after this, seems "POST handler" is something left from previous documentation.